packages/internal/src/index.js

```diff
@@ -8,6 +8,7 @@ export * from './errors.js';
 export * from './js-utils.js';
 export { pureDataMarshaller } from './marshal.js';
 export * from './method-tools.js';
+export * from './panic.js';
```

Comment:
How do I export the `typeof panic` as `Panic` from this module?

Reply:
In `panic.js`, simply `/** @typedef {typeof panic} Panic */`